app/webpacker/components/EditEvents/Modals/EditTimeLimitModal/index.js

```diff
@@ -51,12 +56,12 @@ export default function EditTimeLimitModal({ wcifEvent, wcifRound, disabled }) {
 
   const Title = useMemo(() => `Time limit for ${roundIdToString(wcifRound.id)}`, [wcifRound.id]);
 
-  const hasUnsavedChanges = () => (
+  const hasUnsavedChanges = useCallback(() => (
     !_.isEqual(timeLimit, { centiseconds, cumulativeRoundIds })
-  );
+  ), [timeLimit, cumulativeRoundIds, draftInput]);
```

Comment:
I see that `draftInput` and `centiseconds` are somewhat tied together, but I still don't see how including `draftInput` in this dependency array helps in any way. It should probably be `centiseconds` instead